pkgs/development/python-modules/lxml-html-clean/default.nix

```diff
@@ -26,15 +23,31 @@ buildPythonPackage rec {
 
   dependencies = [ lxml ];
 
-  nativeCheckInputs = [ unittestCheckHook ];
+  nativeCheckInputs = [
+    # Tests should be run with unittest, which pytest is not completely compatible with.
+    # However, only pytest allows us to skip specific tests.
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # Snapshot tests are failing (AssertionError)
```

Comment:
Please link to the upstream issue: https://github.com/fedora-python/lxml_html_clean/issues/24.

Reply:
Done